reader-osm/src/test/java/com/graphhopper/GraphHopperIT.java

```diff
@@ -575,12 +575,12 @@ public void testMonacoVia() {
 
         PathWrapper arsp = rsp.getBest();
         assertEquals(6875.2, arsp.getDistance(), .1);
-        assertEquals(171, arsp.getPoints().getSize());
+        assertEquals(170, arsp.getPoints().getSize());
 
         InstructionList il = arsp.getInstructions();
         assertEquals(30, il.size());
         assertEquals("continue onto Avenue des Guelfes", il.get(0).getTurnDescription(tr));
-        assertEquals("continue onto Avenue des Papalins", il.get(1).getTurnDescription(tr));
+        assertEquals("turn slight left onto Avenue des Papalins", il.get(1).getTurnDescription(tr));
```

Comment:
these changes are probably because by forcing simplification to always keep the closest pillar node to each tower node, we get a bit more resolution on the angles into and out of each intersection.